docs/source/models/concept_models/vivarium_mncnh_portfolio/mncnh_interventions.rst

```diff
@@ -0,0 +1,130 @@
+.. mncnh_interventions:
+
+==================================
+CPAP for treating Preterm with RDS
+==================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - CPAP
+    - Continuous positive airway pressure
+    - 
+  * - RDS
+    - respiratory distress syndrome
+    - 
+
+Intervention Overview
+-----------------------
+
+CPAP is used for the prevention and treatment of respiratory distress syndrome. A recent (2020)Cochrane Review found that CPAP was associated with lower risk of treatment failure (death or use of assisted ventilation) and lower overall mortality with moderate‐certainty evidence.
+
+This section describes how a CPAP intervention can be implemented and calibrated for the :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>`.
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note (ex: is this relationship direct or mediated?)
+  * - Preterm with RDS Mortality Probability
+    - Decrease multiplicatively
+    - Yes
+    - 
+
+Baseline Coverage Data
+++++++++++++++++++++++++
+
+The :ref:`MNCNH Portfolio model <2024_concept_model_vivarium_mncnh_portfolio>` includes data on current coverage levels for the specific locations we are focused on modeling.  We might wish to enhance this by predicting coverage in additional locations, in correlation with other key intervention technologies, based on sources such as existing SPA and SARA data.
+
+Vivarium Modeling Strategy
+--------------------------
+
+We will use the enhanced decision tree below to find the probability of RDS mortality with and without access to CPAP that are logically consistent with the baseline delivery facility rates and baseline CPAP coverage (and population level RDS mortality probability). Since the neonatal model include timesteps for both the early and late neonatal periods, this  alculation must be calculated for both time periods.
+
+We will then add an attribute to each simulant indicating whether the birth occurs at a facility with access to CPAP (which will be dependent on the facility choice, i.e. no access for home births and lower access for BEmONC than CEmONC facilities).
+
+We will then use the conditional probabilities for simulants with and without access to determine which simulants die from RDS.
+
+.. todo::
+
+  include the relative risk of RDS mortality with CPAP here and in the concept model document, without duplicating it (see recent PR to add quickstart docs in the readme for an example of how)
+
+
+Calibration Strategy
```

Comment:
I think it would be useful to add an explanation at the beginning of this section of what the goal is here and why we need to do it. Is this calibrating for baseline coverage, or is it slightly different?